text/src/autogluon/text/automm/models/timm_image.py

```diff
@@ -42,11 +43,13 @@ def __init__(
                 The images are directly averaged and passed to the model.
             - all_logits
                 The logits output from individual images are averaged to generate the final output.
+        pretrained
+            Whether using the pretrained timm models. If pretrained=True, download the pretrained model.
         """
         super().__init__()
         # In TIMM, if num_classes==0, then create_model would automatically set self.model.head = nn.Identity()
         logger.debug(f"initializing {checkpoint_name}")
-        self.model = create_model(checkpoint_name, pretrained=True, num_classes=0)
+        self.model = create_model(checkpoint_name, pretrained=pretrained, num_classes=0)
```

Comment:
If the checkpoint name has a local path, we may need to load the checkpoint after the model is created.